Specs/DateTime/Swedish/TimeZoneExtractor.json

```diff
@@ -0,0 +1,360 @@
+[
+  {
+    "Input": "Boka ett rum åt mig Afghansk tid",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "Afghansk tid",
+        "Type": "timezone",
+        "Start": 20,
+        "Length": 12
+      }
+    ]
+  },
+  {
+    "Input": "Boka ett rum åt mig afghansk tid",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "afghansk tid",
+        "Type": "timezone",
+        "Start": 20,
+        "Length": 12
+      }
+    ]
+  },
+  {
+    "Input": "Jag befinner mig i Amerikansk-Stillahavs normaltid",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "Amerikansk-Stillahavs normaltid",
+        "Type": "timezone",
+        "Start": 19,
+        "Length": 31
+      }
+    ]
+  },
+  {
+    "Input": "Klockan är  Östamerikansk sommartid",
+    "Comment": "This currently doesn't work since diacritics are processed (removed) for all cultures. Line 34 in BaseTimeZoneExtractor.cs. Östamerikansk -> Ostamerikansk.",
+    "NotSupportedByDesign": "javascript, python, java",
+    "NotSupported": "dotnet",
+    "Results": [
+      {
+        "Text": "Östamerikansk sommartid",
```

Comment:
Adding "Ostamerikansk" and similar without the special chars in the dictionary doesn't solve it (i.e. the dictionary contains both versions of the string)? 
We can consider a better solution later.

Reply:
Implemented according to your suggestion.